applications/stepswitch/src/stepswitch_bridge.erl

```diff
@@ -319,6 +319,17 @@ build_bridge(#state{endpoints=Endpoints
             ])
           ,wh_json:get_value(<<"Custom-Channel-Vars">>, JObj, wh_json:new())
          ),
+
+    EndpointFilter = fun(Element) ->
+        case Element of
+            {<<"Outbound-Caller-ID-Number">>, Number} -> false;
+            {<<"Outbound-Caller-ID-Name">>, Name}     -> false;
+            _OtherValues                              -> true
+        end
+    end,
+
+    FmtEndpoints = format_endpoints(Endpoints, Number, JObj, AccountId, EndpointFilter),
+
```

Comment:
try not create these functions inline. create a function (remove_outbound_caller_id ?) and use it.
it also seems you could just drop these keys with delete_keys ?